dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -339,10 +339,11 @@
       # backup:
         # Configure backup store. NOTE:- Use the same configuration on all brokers of this cluster.
 
-        # Set the backup store type. Supported values are [NONE, S3, GCS]. Default value is NONE
+        # Set the backup store type. Supported values are [NONE, S3, GCS, AZURE]. Default value is NONE
         # When NONE, no backup store is configured and no backup will be taken.
         # Use S3 to use any S3 compatible storage (https://docs.aws.amazon.com/AmazonS3/latest/API/Type_API_Reference.html).
         # Use GCS to use Google Cloud Storage (https://cloud.google.com/storage/)
+        # Use AZURE to use Azure Storage (https://learn.microsoft.com/en-us/azure/storage/)
```

Comment:
:wrench: Let's clarify that it is only Azure **Blob** Storage that we support: https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction